site/content/docs/5.2/components/progress.md

```diff
@@ -49,6 +49,13 @@ Boosted provides a handful of [utilities for setting width]({{< docsref "/utilit
 
 Add labels to your progress bars by placing text within the `.progress-bar`.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant should not be used because it does not respect the Orange Design System specifications.
```

Comment:
So how do we display the information based on ODS guidelines? Is there a link somewhere?

Reply:
Yes just below. See line 56.

Reply:
No the value as a label isn't displayed.

Reply:
Resolved ?